test/jdk/java/time/test/java/time/format/TestFractionPrinterParser.java

```diff
@@ -79,7 +79,7 @@
 /**
  * Test FractionPrinterParser.
```

Comment:
OK, then I'd add some comments that the test covers `NanoPrinterParser` as well.

Reply:
Ok, done.